File: Codingbat/Java/Array-1/reverse3.java
public int[] reverse3(int[] nums) {
  int [] reverse = new int [nums.length]; 
  int idx = 0;
  for(int i=nums.length-1; i>=0; i--){
    reverse[i] = nums[idx];
    idx++;
  }
  return reverse;
}

File: Codingbat/Python/List-1/reverse3.py
def reverse3(nums):
  # solution 1 (uncomment) 
  # return nums[::-1]
  
  # solution 2
  # output = []
  # for i in range(len(nums)-1, -1, -1): 
  #   output.append(nums[i]) 
  # return output 
  
  
  # solution 3 
  ans = [] 
  for num in reversed(nums): 
    ans.append(num)
  return ans
    
  
